google-signin.html

```diff
@@ -7,11 +7,12 @@
 <link rel="import" href="../paper-material/paper-material.html">
 <link rel="import" href="../iron-flex-layout/iron-flex-layout-classes.html">
 <link rel="import" href="google-icons.html">
+<link rel="import" href="google-signin-styles.html">
 
 <dom-module id="google-signin">
-  <link rel="import" type="css" href="google-signin.css">
   <template>
-    <style is="custom-style" include="iron-positioning"></style>
```

Comment:
@jeffposnick  `is="custom-style"` isn't necessary/shouldn't be used inside of custom elements, so I removed it with this PR as well.